kubernetes/istio_details_service.go

```diff
@@ -266,6 +268,27 @@ func (in *IstioClient) GetQuotaSpecBinding(namespace string, quotaSpecBindingNam
 	return quotaSpecBinding.DeepCopyIstioObject(), nil
 }
 
+// UpdateIstioObject updates an Istio object from either config api or networking api
+func (in *IstioClient) UpdateIstioObject(api, namespace, resourceType, name, jsonPatch string) (IstioObject, error) {
+	log.Infof("UpdateIstioObject input: %s / %s / %s / %s", api, namespace, resourceType, name)
```

Comment:
If it's for auditing (I believe it is), shouldn't we also log `jsonPatch` ?

Reply:
This is just for informational.
Auditing logging will have a specific format and I planned to address it on KIALI-1771.
(As it could have more config logic to decide the verbosity, etc, and I didn't plan to do it on this one).

But yeah, comment is valid.